recipes-kernel/linux/linux-mainline/openvario-57-lvds_DS2.dts

```diff
@@ -0,0 +1,38 @@
+#include "openvario-common.dts"
+
+&panel {
+    compatible = "openvario,lvds57", "panel-lvds";
+    width-mm = <117>;
+    height-mm = <89>;
+    data-mapping = "jeida-18";
+
+    panel-timing {
+        /* 640x480 @60Hz */
+        clock-frequency = <25000000>;
+        hactive = <640>;
+        vactive = <480>;
+        hsync-len = <1>;
+        hfront-porch = <2>;
+        hback-porch = <157>;
+        vsync-len = <1>;
+        vfront-porch = <2>;
+        vback-porch = <42>;
+    };
+};
```

Comment:
Is this a complete copy of `openvario-57-lvds.dts`?

Reply:
No, UART configuration has changed.